python/pyfory/_registry.py

```diff
@@ -485,6 +488,15 @@ def _create_serializer(self, cls):
                 serializer = DataClassSerializer(self.fory, cls)
             elif issubclass(cls, enum.Enum):
                 serializer = EnumSerializer(self.fory, cls)
+            elif hasattr(cls, "__getstate__") and hasattr(cls, "__setstate__"):
+                # Use StatefulSerializer for objects that support __getstate__ and __setstate__
+                # But exclude certain types that have incompatible state methods
+                module_name = getattr(cls, "__module__", "")
+                if module_name.startswith("pandas."):
```

Comment:
Could we create an issue to address this in the future?

Reply:
What do you want to do differently?